tests/onnxruntime/test_modeling.py

```diff
@@ -3120,6 +3123,9 @@ def test_generate_utils(self, test_name: str, model_arch: str, use_cache: str):
 
     @parameterized.expand(SUPPORTED_ARCHITECTURES)
     def test_merge_from_transformers_and_save(self, model_arch):
+        if model_arch == "encoder-decoder":
+            self.skipTest("encoder-decoder model type with use_merged=True is not supported for bert as a decoder")
```

Comment:
Does this mean that encoder-decoder is not tested for merged onnx?

Reply:
The test uses `bert-bert` model for testing, so only `use_cache=False` is used, which cannot work with `use_merged=True`

Reply:
My question remains. There are some encoder-decoder that support past KV. I am wondering if this is tested anywhere.

Reply:
Ok got it. There was no suitable model to add in testing for such a model type. I will create a custom model and add a new pr now